draft-ietf-jsonpath-base.md

```diff
@@ -517,6 +521,18 @@ This is the concatenation of three lists, two of length one containing `0`, `1`,
 As a consequence of this approach, if any of the selectors selects no nodes,
 then the whole JSONPath selects no nodes.
 
+The current item selector `@` when used outside a filter behaves as the document
+root selector unless a child node is given to the evaluator.
+When a child node is used as current item, evaluator must select it as starting
+node instead of the document root when an expression starting with `@` is
+evaluated.
+
+When a child node is used as current item, the current item selector `@` selects
```

Comment:
I'm not sure who's working with English as a first language, so please don't take offense, but this could use some grammatical massaging, specifically the use of the word "the" in a few places.

Reply:
>  but this could use some grammatical massaging

It is not my first language, so any advice or feedback is welcome

Anyway I think that I'll remove this and just keep:
```
The current item selector `@` can be used to represent the current item being
evaluated. The initial current item defaults to the document root unless a
subdocument is given to the evaluator as initial current item.
```

as described in the other conversation, if you agree too that "`When a child node is used as current item, the`"... is just redundant we can remove it and close this conversation since we don't need rephrasing it.